tests/integration/test_charm.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+from pathlib import Path
+
+import pytest
+import yaml
+from pytest_operator.plugin import OpsTest
+from tests.integration.helpers import pull_content_from_unit_file
+from pymongo import MongoClient
+
+logger = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+APP_NAME = METADATA["name"]
+
+
+@pytest.mark.abort_on_fail
+async def test_build_and_deploy(ops_test: OpsTest) -> None:
+    """Build and deploy one unit of MongoDB"""
+    my_charm = await ops_test.build_charm(".")
+    await ops_test.model.deploy(my_charm)
+    await ops_test.model.wait_for_idle()
+
+
+@pytest.mark.abort_on_fail
+async def test_status(ops_test: OpsTest) -> None:
+    """Verifies that the application and unit are active"""
+    await ops_test.model.wait_for_idle(
+        apps=[APP_NAME], status="active", timeout=1000
+    )
+    assert len(ops_test.model.applications[APP_NAME].units) == 1
+    assert ops_test.model.applications[APP_NAME].units[0].workload_status == "active"
```

Comment:
You shouldn't need this line, since you're already checking the status is active in line 32.